src/main/scala/dotty/dokka/tasty/ClassLikeSupport.scala

```diff
@@ -13,61 +13,93 @@ trait ClassLikeSupport:
 
   def parseClass(classDef: reflect.ClassDef)(using ctx: Context): DClasslike =
     val parents = for
-      parentTree <- classDef.parents // We assume here that order is correct
+      parentTree <- classDef.parents if isValidPos(parentTree.pos)  // We assume here that order is correct
```

Comment:
We could use an explanatory comment or a link to a file that requires to check if pos of a parent is valid. Is this evident from Git history?